src/main/scala/viper/gobra/frontend/info/implementation/resolution/AmbiguityResolution.scala

```diff
@@ -99,14 +100,15 @@ trait AmbiguityResolution { this: TypeInfoImpl =>
 
     case n: PInvoke =>
       exprOrType(n.base) match {
-        case Right(t) => Some(ap.Conversion(t, n.args))
+        case Right(t) if n.args.length == 1 => Some(ap.Conversion(t, n.args.head))
```

Comment:
why did you do this change?

Reply:
According to the Go spec,
> An explicit conversion is an expression of the form T(x) where T is a type and x is an expression that can be converted to type T.

In other words, a conversion must have exactly one argument. Before, `ap.Conversion` expected an argument of type `Vector[Expr]` and we had a second check somewhere that it had exactly one value. I don't see a point in having all this complexity. Instead, we can just make invalid instances of ap.Conversion unrepresentable